### ISSUE TITLE
Reaching Admin page appears to be programming problem with Index.php

### ISSUE THREAD

**Issue by ghost** [NathanGibbs3/BASE#94]
Item | Description
-----|-------------
**Link to example**: | Optional link to CI build.
**Build Environment**: | ( Travis-CI or Local )
**PHP Version**: | 7.4.14 
Apache 2.4.26 x64
**PHPUnit Version**: |
**Similar Issues:** | #142 
**Depends on Issue(s):** | #144 
**Dependency Type:** | Hard
**Misc Info**: | Trying to reach the Admin page.
---
**Expected Behavior**:
**Current Behavior**:
http://winids/admin/index.php

Internal server Error...

This error (HTTP 500 Internal Server Error) means that the website you are visiting had a server problem which prevented the webpage from displaying.

Works fine with other .php files in the same folder.

**Solution**: Not sure.
Any thoughts on how to fix this.

**Comment by NathanGibbs3** [comment]
Information from the web server's error log may be helpful in diagnosing this issue.
Thanks. :smile:

**Comment by mesteele101** [comment]
If the database, and database tables are manually created, and NOT created by BASE, will that cause a problem?

**Comment by mesteele101** [comment]
So to start off with I have never used anything in the Admin function to setup BASE. I manually setup the ALL the database options and it works. With PHP 5 BASE I have no issues accessing the Admin function in the BASE console.

What I did to test this was to rename the php folder to php-5 and dissolve the contents of PHP 7 to PHP. I moved all original configuration from the PHP-5 php.ini file to the new PHP 7 php.ini. The only change was from extension=php_mysql.dll to extension=php_mysqli.dll.

For BASE I did the same thing.

I deleted the error.log for Apache and restarted.

BASE starts up as normal. I go to the Admin function and I get:

This error (HTTP 500 Internal Server Error) means that the website you are visiting had a server problem which prevented the webpage from displaying.

See the attached error log.
[error.log](https://github.com/NathanGibbs3/BASE/files/6409824/error.log)
